src/index.ts

```diff
@@ -38,11 +39,28 @@ export interface IClientConfig {
   assignmentLogger: IAssignmentLogger;
 }
 
-export { IAssignmentEvent, IAssignmentLogger } from './assignment-logger';
-export { IEppoClient } from './client/eppo-client';
+export { IAssignmentLogger, IAssignmentEvent, IEppoClient } from '@eppo/js-client-sdk-common';
 
 let poller: IPoller;
-let clientInstance: IEppoClient;
+let clientInstance: IEppoServerClient;
+
+interface IEppoServerClient extends EppoServerClient {
+  /**
+   * Used to manually stop the polling of Eppo servers.
+   */
+  stopPolling(): void;
+}
+
+export class EppoServerClient extends EppoClient implements IEppoClient {
```

Comment:
Wondering if I should name this `EppoNodeClient` instead?

Reply:
Yeah that would be a better name